static/docs/user-guide/running-dvc-on-windows.md

```diff
@@ -1,9 +1,9 @@
 # Running DVC on Windows
 
-There can be few performance issues while running DVC on Windows by default.
-They are mostly related to NTFS (a file system that the Windows uses)
-characteristics and Windows built-in security mechanisms. Below are some
-workarounds that can help avoid these performance penalties:
+There can different issues while running DVC on Windows. Some, for example, have
```

Comment:
`can be`

I would mention that performance is the biggest problem

Reply:
Done in 192b092